### ISSUE TITLE
exp: compare Llama to Qwen3

### ISSUE THREAD

**Issue by eric-czech** [Open-Athena/MarinFold#29]
Compare Llama vs equivalent Qwen3 at 100M on the **m11** size-proportional protein-docs mixture (Open-Athena/MarinFold#11).  Use LR from Llama tuning for both to advantage Llama.

### Setup

- **Model:** 100M (h=768, l=12, dff=3072, heads=12, kv=4). Llama config is [`protein_llama_100m`][model]; the Qwen3 mirror keeps those dims and adds QK-norm (Qwen3 default; sliding window off).
- **Data:** m11 — H/M/L train splits at weights 0.31 / 0.26 / 0.43 (proportional to per-split sizes), loss-masked to distance-bin tokens.
- **Optimizer / schedule:** batch=128, seq=8192, 4104 steps (~4.3B tokens), warmup=0.1, WSD-style linear decay over the trailing 0.2 fraction. LR=9.33e-4 and β₂=0.98 adapted from [`train_protein_1_5b_distance_masked.py`][lineage] via the sqrt-batch / inverse-hidden scaling.
- **Eval:** cd-val + H/M/L (val, test) at 16 batches × 128 examples per split (~2k examples / 16.8M tokens each).

### Code

[`experiments/protein/exp29_arch_sweep.py`][sweep] on branch `eac/plm-exp29`.

### Runs

W&B group [`exp29-arch`][group] — [llama][llama] · [qwen3][qwen3].

[sweep]: https://github.com/marin-community/marin/blob/26fb45734/experiments/protein/exp29_arch_sweep.py
[model]: https://github.com/marin-community/marin/blob/26fb45734/experiments/protein/train_protein_100m_distance_masked.py
[lineage]: https://github.com/marin-community/marin/blob/26fb45734/experiments/protein/train_protein_1_5b_distance_masked.py
[group]: https://wandb.ai/eric-czech/marin/groups/exp29-arch
[llama]: https://wandb.ai/eric-czech/marin/runs/prot-exp11-dm-arch-100m-4.303B-m11-llama-lr9.3e-4-v1-49838a
[qwen3]: https://wandb.ai/eric-czech/marin/runs/prot-exp11-dm-arch-100m-4.303B-m11-qwen3-lr9.3e-4-v1-c5b1e1

**Comment by eric-czech** [comment]
## Results

![Llama vs Qwen3 — val loss by trial @ step 4103](https://gist.githubusercontent.com/eric-czech/0f5b657aa923e5ac2e92df1d75f93386/raw/8c8fd8d72f8c33e466ae0ea91cc0a0dde67354ca/llama_vs_qwen3.png)

<details><summary>Data</summary>

```
variant_id,variant_name,metric,value,step_used,ref_step,run_name,run_state
llama,Llama,cd-val-unmasked,10.898519515991213,4103,4103,prot-exp11-dm-arch-100m-4.303B-m11-llama-lr9.3e-4-v1-49838a,finished
llama,Llama,cd-val,1.4080580472946167,4103,4103,prot-exp11-dm-arch-100m-4.303B-m11-llama-lr9.3e-4-v1-49838a,finished
llama,Llama,high-test,1.403142333030701,4103,4103,prot-exp11-dm-arch-100m-4.303B-m11-llama-lr9.3e-4-v1-49838a,finished
llama,Llama,high-val,1.4080580472946167,4103,4103,prot-exp11-dm-arch-100m-4.303B-m11-llama-lr9.3e-4-v1-49838a,finished
llama,Llama,low-test,1.4459443092346191,4103,4103,prot-exp11-dm-arch-100m-4.303B-m11-llama-lr9.3e-4-v1-49838a,finished
llama,Llama,low-val,1.4692116975784302,4103,4103,prot-exp11-dm-arch-100m-4.303B-m11-llama-lr9.3e-4-v1-49838a,finished
llama,Llama,medium-test,1.4208520650863647,4103,4103,prot-exp11-dm-arch-100m-4.303B-m11-llama-lr9.3e-4-v1-49838a,finished
llama,Llama,medium-val,1.4427366256713867,4103,4103,prot-exp11-dm-arch-100m-4.303B-m11-llama-lr9.3e-4-v1-49838a,finished
llama,Llama,train,1.5013831853866575,4103,4103,prot-exp11-dm-arch-100m-4.303B-m11-llama-lr9.3e-4-v1-49838a,finished
qwen3,Qwen3 (+QK-norm),cd-val-unmasked,10.753999710083008,4103,4103,prot-exp11-dm-arch-100m-4.303B-m11-qwen3-lr9.3e-4-v1-c5b1e1,finished
qwen3,Qwen3 (+QK-norm),cd-val,1.327430248260498,4103,4103,prot-exp11-dm-arch-100m-4.303B-m11-qwen3-lr9.3e-4-v1-c5b1e1,finished
qwen3,Qwen3 (+QK-norm),high-test,1.3210086822509766,4103,4103,prot-exp11-dm-arch-100m-4.303B-m11-qwen3-lr9.3e-4-v1-c5b1e1,finished
qwen3,Qwen3 (+QK-norm),high-val,1.327430248260498,4103,4103,prot-exp11-dm-arch-100m-4.303B-m11-qwen3-lr9.3e-4-v1-c5b1e1,finished
qwen3,Qwen3 (+QK-norm),low-test,1.3636815547943115,4103,4103,prot-exp11-dm-arch-100m-4.303B-m11-qwen3-lr9.3e-4-v1-c5b1e1,finished
qwen3,Qwen3 (+QK-norm),low-val,1.386289358139038,4103,4103,prot-exp11-dm-arch-100m-4.303B-m11-qwen3-lr9.3e-4-v1-c5b1e1,finished
qwen3,Qwen3 (+QK-norm),medium-test,1.337774634361267,4103,4103,prot-exp11-dm-arch-100m-4.303B-m11-qwen3-lr9.3e-4-v1-c5b1e1,finished
qwen3,Qwen3 (+QK-norm),medium-val,1.3602715730667114,4103,4103,prot-exp11-dm-arch-100m-4.303B-m11-qwen3-lr9.3e-4-v1-c5b1e1,finished
qwen3,Qwen3 (+QK-norm),train,1.4178893566131592,4103,4103,prot-exp11-dm-arch-100m-4.303B-m11-qwen3-lr9.3e-4-v1-c5b1e1,finished
```

</details>

Qwen3 is a clear, consistent win over Llama across every val split.

Raw results: [exp29_arch_results.zip](https://github.com/user-attachments/files/28325492/exp29_arch_results.zip)
